custom/icds/location_reassignment/parser.py

```diff
@@ -469,20 +470,36 @@ def __init__(self, domain, workbook):
 
     def parse(self):
         errors = []
+        site_codes = set([ws.title for ws in self.workbook.worksheets])
+        location_ids = {loc.site_code: loc.location_id
+                        for loc in SQLLocation.objects.filter(domain=self.domain, site_code__in=site_codes)}
```

Comment:
Copied over from `HouseholdReassignmentParser` above.
DRYed up in later commit.